README.md

```diff
@@ -3,4 +3,4 @@ A Document Management System
 
 [![Code Climate](https://codeclimate.com/github/vynessa/dman/badges/gpa.svg)](https://codeclimate.com/github/dman)
 [![Travis Build](https://img.shields.io/travis/vynessa/dman/develop.svg)](https://travis-ci.org/vynessa/dman)
-[![Coverage Status](https://coveralls.io/repos/github/vynessa/dman/badge.svg?branch=master)](https://coveralls.io/github/vynessa/dman?branch=master)
+[![Coverage Status](https://coveralls.io/repos/github/vynessa/dman/badge.svg?branch=feature%2F%23149869630%2Fusers-endpoint)](https://coveralls.io/github/vynessa/dman?branch=feature%2F%23149869630%2Fusers-endpoint)
```

Comment:
I understand the reason for this update, but I think you should update it to `develop`, and also set your default branch to `develop`, that would make the flow more seamless amongst other things.